src/lib/data/banners/events/3.5.json

```diff
@@ -69,6 +69,71 @@
 					}
 				}
 			}
+		},
+		{
+			"phase": 2,
+			"banners": {
+				"standardVersion": 3,
+				"events": {
+					"item": [
+						{
+							"name": "the-transcendent-one-returns-2",
+							"character": "shenhe",
+							"buttonBoxPosition": {
+								"w": 160,
+								"t": 22
```

Comment:
Because this character banner already existed before, it's better to use the same settings I've applied.

https://github.com/AguzzTN54/Genshin-Impact-Wish-Simulator/blob/36fe7a881a311dd75cd37df1ffff6e7c191becc9/src/lib/data/banners/events/2.4.json#L12-L17